addons/im_livechat/views/im_livechat_channel_views.xml

```diff
@@ -234,7 +234,7 @@
                     <sheet>
                         <group>
                             <field name="action" widget="radio"/>
-                            <field name="regex_url" placeholder="e.g. /page/contactus"/>
```

Comment:
Boarf, commit message is way too elaborate for surch a tiny change.

"Fixed URL in placeholders to march existing "contactus" page URL".

And could also be squashed with the 2 first ones.